src/components/UI/Display/Cluster/ClusterIDLabel/index.js

```diff
@@ -48,20 +48,38 @@ const ClusterIDLabel = ({ clusterID, copyEnabled }) => {
     setClipboardContent(clusterID);
   };
 
+  const realClusterID = clusterID;
+
+  var label = realClusterID.substring(0, 5);
+  if (label !== realClusterID) {
+    label = `${realClusterID.substring(0, 4)}…`;
+  }
+
+  const labelComponent = (
+    <Label clusterID={realClusterID}>
+      <OverlayTrigger
+        overlay={
+          <Tooltip id='idtooltip'>
+            <>Cluster ID: {realClusterID}</>
+          </Tooltip>
+        }
+        placement='top'
+      >
+        <span aria-label={realClusterID}>{label}</span>
```

Comment:
There's no need for a new variable here 👍🏻 

```suggestion
  let label = clusterID.substring(0, 5);
  if (label !== clusterID) {
    label = `${clusterID.substring(0, 4)}…`;
  }

  const labelComponent = (
    <Label clusterID={clusterID}>
      <OverlayTrigger
        overlay={
          <Tooltip id='idtooltip'>
            <>Cluster ID: {clusterID}</>
          </Tooltip>
        }
        placement='top'
      >
        <span aria-label={clusterID}>{label}</span>
```